### PR TITLE
CEDS-1793 Add visually hidden messages to change buttons

### PR DESCRIPTION
**Pull request by PauloPortugal**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -6,6 +6,10 @@ site.continue = Continue
 site.back = Back
 site.save_and_continue = Save and continue
 site.change = Change
+site.change.hint.disassociate.ucr = Change DUCR dissociation
+site.change.hint.associate.ucr = Change DUCR association
+site.change.hint.associate.mucr = Change MUCR association
+site.change.hint.shut.mucr = Change MUCR
```
Comment (MaciejRewera) [inline conf/messages.en]: I would rather use `disassociateUcr` etc. Using dot as a space does not fit in here.

File: app/views/shut_mucr_summary.scala.html [hunk 2]
```diff
@@ -41,7 +41,10 @@ <h2 id="shutMucr-header">@messages("shutMucr.summary.header")</h2>
                     <td id="shutMucr-type">@messages("shutMucr.summary.type")</td>
                     <td id="shutMucr-mucr">@mucr</td>
                     <td class="text-right">
-                        <a id="shutMucr-change" href="@routes.ShutMucrController.displayPage()">@messages("site.edit")</a>
+                        <a id="shutMucr-change" href="@routes.ShutMucrController.displayPage()">
+                            <span aria-hidden="true">@messages("site.edit")</span>
+                            <span class="visuallyhidden">@{messages("site.change.hint.disassociate.ucr")}</span>
```
Comment (MaciejRewera) [inline app/views/shut_mucr_summary.scala.html]: Not sure, but I think it should be `site.change.hint.shut.mucr`

File: app/views/associate_ucr_summary.scala.html [hunk 3]
```diff
@@ -55,7 +58,10 @@ <h2 id="master-consignment">@messages("associate.ucr.summary.masterConsignment")
                     <td class="associate_ucr__to" id="mucr-type">@messages("associate.ucr.summary.kind.mucr")</td>
                     <td id="mucr-reference">@mucr</td>
                     <td class="text-right associate_ucr__change">
-                        <a id="mucr-change" href="@consolidations.routes.MucrOptionsController.displayPage()">@messages("site.change")</a>
+                        <a id="mucr-change" href="@consolidations.routes.MucrOptionsController.displayPage()">
```
Comment (MaciejRewera) [inline app/views/associate_ucr_summary.scala.html]: Could you make id in a similar fashion as the previous one?

File: conf/messages.en [hunk 4]
```diff
@@ -6,6 +6,10 @@ site.continue = Continue
 site.back = Back
 site.save_and_continue = Save and continue
 site.change = Change
+site.change.hint.disassociate.ucr = Change DUCR dissociation
+site.change.hint.associate.ucr = Change DUCR association
+site.change.hint.associate.mucr = Change MUCR association
```
Comment (PatrykRudnicki) [inline conf/messages.en]: If we're splitting associate to two different messages (not using one and passing argument. The first one should be probably `ducr` not `ucr`.

```
site.change.hint.associate.ducr = Change DUCR association
site.change.hint.associate.mucr = Change MUCR association
```
or
`site.chage.hint.associate.ucr = Change {0} association`

File: app/views/shut_mucr_summary.scala.html (hunk 2 above)
Comment (PauloPortugal) [reply app/views/shut_mucr_summary.scala.html]: Correct. Copy paste error.

File: app/views/associate_ucr_summary.scala.html (hunk 3 above)
Comment (PauloPortugal) [reply app/views/associate_ucr_summary.scala.html]: Done.

File: app/views/shut_mucr_summary.scala.html (hunk 2 above)
Comment (PauloPortugal) [reply app/views/shut_mucr_summary.scala.html]: Fixed

File: conf/messages.en (hunk 1 above)
Comment (PauloPortugal) [reply conf/messages.en]: I think `disassociate.ucr` reads better than `disassociateUcr`. I have also used the same ID for the `movements` service.

File: conf/messages.en (hunk 4 above)
Comment (PauloPortugal) [reply conf/messages.en]: Good suggestion. Done.

**Pull request merged**